packages/api-i18n/src/plugins/service.ts

```diff
@@ -29,6 +29,14 @@ export default () => [
                 return localesCache;
             }
 
+            const urlRegEx = /^(https?:\/\/)((([a-z\d]([a-z\d-]*[a-z\d])*)\.)+[a-z]{2,}|((\d{1,3}\.){3}\d{1,3}))(\:\d+)?(\/[-a-z\d%_.~+]*)*(\?[;&a-z\d%_.~+=-]*)?(\#[-a-z\d_]*)?$/i;
+
+            if (!urlRegEx.test(process.env.I18N_API_URL)) {
```

Comment:
Please use our own [validation library](https://github.com/webiny/webiny-js/tree/master/packages/validation) for checking URLs.